calculator/src/test/java/com/market/banica/calculator/controllerTests/CalculatorControllerTest.java

```diff
@@ -44,13 +46,15 @@ void getRecipe() throws Exception {
         String product = "baklava";
         int quantity = 100;
 
-        RecipeDTO dummyRecipe = new RecipeDTO();
-        dummyRecipe.setIngredients(new HashSet<>());
+        List<ProductDto> productDtoList = new ArrayList<>();
```

Comment:
population of the list could be extracted in a separate method and all the strings should be in a constant

Reply:
Fixed